### PR TITLE
Bump/alt/cultuur connect

### PR DESCRIPTION
**Pull request by MoretS**
Bump current 0.18 decidim version to 0.23-stable from BCN repo with new decidim modules enabled

### CODE REVIEW

File: config/locales/decidim-awesome/fr.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+              allow_images_in_markdown_editor: Permettre à l'éditeur Markdown d'envoyer des images par glisser/déposer ou par copier/coller.
+              allow_images_in_proposals: Permettre l'envoi d'image dans l'éditeur de proposition, pour tous les utilisateurs.
+              allow_images_in_small_editor: Ajouter une icône d'envoi d'image dans tous les éditeurs WYSIWYG avec une barre d'outil minimale.
+              drag_and_drop_supported: Support du glisser/déposer lors de l'envoi d'image. Les images seront envoyées sur le serveur et traitées comme des ressources externes (sans encodage Base64)
+              use_markdown_editor: Remplace l'éditeur WYSIWYG Quill par un éditeur Markdown coté admin et un moteur de rendu HTML coté utilisateur (le texte sera stocké au format Markdown)
+          rich_text_editor_in_public_views: "NOTE: l'éditeur de texte riche est activé pour les participants. Cette option sera donc sans effet. Utilisez plutôt les \"editors hacks\" pour ajouter des images dans les textes des propositions"
+          show:
+            title: Tweaks for %{setting}
```
Comment (Quentinchampenois) [inline config/locales/decidim-awesome/fr.yml:47]: Hi @MoretS,

Is it voluntary to keep this key in English ? 

```
            title: Tweaks for %{setting}
```

**Review by Quentinchampenois — APPROVED** [review]

File: config/locales/decidim-awesome/fr.yml (hunk 1 above)
Comment (armandfardeau) [reply config/locales/decidim-awesome/fr.yml:47]: +1

**Review by armandfardeau — CHANGES_REQUESTED** [review]
Fix the missing translation and we are to go

**Review by armandfardeau — APPROVED** [review]

**Comment by armandfardeau** [comment]
Are we good to go? @MoretS

**Review by moustachu — APPROVED** [review]

**Review by armandfardeau — APPROVED** [review]

**Pull request merged**